client/js/components/document/DpMapSettingsPreview.vue

```diff
@@ -72,7 +72,7 @@
             v-tooltip="Translator.trans(link.tooltipContent)"
             class="o-link"
             :class="{'color-status-complete-text': link.done()}"
-            :data-cy="`gisLayerLink:${index}`"
+            :data-cy="`gisLayerLink:${dataCyFormating(Translator.trans(link.label))}`"
```

Comment:
why not like so?
```suggestion
            :data-cy="`gisLayerLink:${link.label}`"
```
then we don'T have to transform the text.
And using a transated Text as hook is unstable anyhow. that can change and vary over different projects quite often.

Reply:
Yes you are right, thank you!

Reply:
https://github.com/demos-europe/demosplan-core/pull/3665/commits/eb968af9414b7abc99de14804ae5d7ecbe723373